src/main/java/cz/muni/ics/oidc/ToOidcSynchronizer.java

```diff
@@ -368,6 +368,11 @@ private void setRefreshTokens(MitreidClient c, Map<String, PerunAttributeValue>
                 c.getGrantTypes().add(GRANT_REFRESH_TOKEN);
                 c.setClearAccessTokensOnRefresh(true);
                 c.setReuseRefreshToken(false);
+                PerunAttributeValue reuseTokens = attrs.getOrDefault(perunAttrNames.getReuseRefreshTokens(), null);
+                if (reuseTokens != null) {
+                    c.setReuseRefreshToken(reuseTokens.valueAsBoolean());
+                } else {
+                }
```

Comment:
empty `else` block?

Reply:
Sorry, I have created the PR before pushing the final changes :)